types/lunr/index.d.ts

```diff
@@ -127,9 +127,14 @@ declare namespace lunr {
          * All fields should be added before adding documents to the index. Adding fields after
          * a document has been indexed will have no effect on already indexed documents.
          *
+         * Fields can be boosted at build time. This allows terms within that field to have more
+         * importance when ranking search results. Use a field boost to specify that matches
+         * within one field are more important than other fields.
+         *
          * @param field - The name of a field to index in all documents.
+         * @param attributes - Optional attributes associated with this field.
          */
-        field(field: string): void;
+        field(field: string, attributes?: object): void;
```

Comment:
Used `object` due to attributes being set as a value of `_fields` which is just an object.

https://github.com/olivernn/lunr.js/blob/master/lunr.js#L2423
```javascript
this._fields[fieldName] = attributes || {}
```
https://github.com/olivernn/lunr.js/blob/master/lunr.js#L2360
```javascript
lunr.Builder = function () {
  this._ref = "id"
  this._fields = Object.create(null)
  this._documents = Object.create(null)
```